src/routes/robots.txt/+server.ts

```
@@ -1,7 +1,36 @@
+Disallow: /*?trk=
+Disallow: /*&utm_
+Disallow: /*&ref=
+Disallow: /*&adobe_mc=
+Disallow: /*&trk=
+
+# Block internal/auth pages
+Disallow: /console/login
+Disallow: /console/register
+Disallow: /v1/
+
+# Block all versioned docs references; only \"cloud\" should be indexable
```

ESLint

[error] Unnecessary escape character: \".

(no-useless-escape)